tests/core_tests/loki_tests.cpp

```diff
@@ -1055,14 +1055,15 @@ bool loki_name_system_expiration::generate(std::vector<test_event_entry> &events
   gen.add_mined_money_unlock_blocks();
 
   lns_keys_t miner_key = make_lns_keys(miner);
-  if (lns::mapping_type_allowed(gen.hardfork(), lns::mapping_type::lokinet))
+  if (lns::mapping_type_allowed(gen.hardfork(), lns::mapping_type::lokinet_1year))
```

Comment:
Looks like all the tests now are just using 1year regs.  Perhaps this run all the same tests but in a loop to go through the different registration lengths (without adding much code)?

Reply:
Changed it to test all years